readme.md

```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# React Capstone Project - COVID-19 DATA
+
+> This capstone project is a Covid-19 tracker which displays the most recent COVID-19 statistics around the globe from the [Narrativa API](https://covid19tracking.narrativa.com/index_en.html). It is also a mobile-first application built with React, Redux, JavaScript, Bootstrap, API
+
+
+## Built With
+
+- JavaScript
+- React
+- Redux
+- React Bootsrap
+
+## Live Demo
+[Live Demo - Netlify](https://cvd-trak.netlify.app/)
+
+## Getting Started
+
+To get a local copy follow these simple example steps.  
+
+1. Open the terminal window and clone the repository using this command:  
+`git@github.com:41hulk/metric-app.git` 
+
+2. Change the directory to the react-bookstore directory  
+`cd react-capstone-project`  
+3. Install the project's dependencies by running this command:   
+`npm install`   
+4. Run this command to see the page `npm start`  
+5. To see the code open the project in the code editor of your preference.
+
+<br>
+
+>OBS.: The data is from the [Narrativa API](https://covid19tracking.narrativa.com/index_en.html). For more information, please check its documentation.
+
+<br>
+
+## Author
+👤 **Ntare Guy**
+- GitHub: [@Ntare Guy](https://github.com/41hulk)
+- Twitter: [@Ntare Guy](https://twitter.com/ntare-guy)
+- Linkedin: [@Ntare Guy](https://www.linkedin.com/in/ntare-guy/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/41hulk/metric-app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
```

Comment:
- [ ] Please, consider that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Thanks for the comment @elmar8287